.github/workflows/coveralls.yml

```diff
@@ -0,0 +1,28 @@
+name: Coverage
+
+# Make this manual for now.
+# It could also be scheduled using
+# on:
+#   schedule:
+#    -cron: '0 0 * * SAT'
+on:
+  workflow_dispatch
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: config
+      run: CC=gcc ./config --debug --coverage no-asm enable-rc5 enable-md2 enable-ssl3 enable-nextprotoneg enable-ssl3-method enable-weak-ssl-ciphers enable-zlib enable-ec_nistp_64_gcc_128 no-shared enable-buildtest-c++ enable-external-tests -DPEDANTIC -DFUZZING_BUILD_MODE_UNSAFE_FOR_PRODUCTION && perl configdata.pm --dump
```

Comment:
I've discovered that lines can be broken and still remain the same value.  There's not even a need to mark it!  So if you want to avoid unwieldy long lines, just break it up:

```
      run: CC=gcc ./config --debug --coverage no-asm enable-rc5 enable-md2 enable-ssl3
           enable-nextprotoneg enable-ssl3-method enable-weak-ssl-ciphers enable-zlib
           enable-ec_nistp_64_gcc_128 no-shared enable-buildtest-c++ enable-external-tests
           -DPEDANTIC -DFUZZING_BUILD_MODE_UNSAFE_FOR_PRODUCTION
           && perl configdata.pm --dump

Reply:
One line can be directly copied to shell...

Reply:
Oh, valid point!